### ISSUE TITLE
Multitenant templates for Django 1.8

### ISSUE THREAD

**Issue by julialpz** [bernardopires/django-tenant-schemas#289]
Hello!

I am trying to use multitenant templates with django 1.8. However, in template_loaders.py you are using 3 methods from django.template.loader that seem to have been removed in django 1.8 (get_template_from_string definitely has been (https://docs.djangoproject.com/en/1.8/ref/templates/upgrading/#get-template-from-string), I can't find anything on the other 2). Do you plan on updating that?

Thanks!
Julia

**Comment by fablet** [comment]
There is a spin off from this project with more django 1.8 support at https://github.com/tomturner/django-tenants
I didn't like the extra steps for working with the domain_url in that package though so I just copied out the template_loaders.py from that branch and dumped it into my local middleware.
With this I stopped getting the django 1.8 errors and it works correctly to find my tenant templates, but for some odd reason it will show the changes in different base.htmls but not the index.htmls inside of it.
Once you get this up and running I would be interested to know if you have the same experience.
